css/style.css

```diff
@@ -0,0 +1,903 @@
+}
+
+.featured-img__style {
+    width: 371px;
+    cursor: pointer;
+    transition: .3s;
+}
+
+.featured-img__style:hover {
+    transition: .3s;
+    transform: scale(1.03);
+}
+
+.small-team {
+    min-height: 1360px;
+    margin: 0 auto;
+    padding-top: 175px;
+    background-color: #f9fbfb;
+}
+
+.small-team__text {
+    text-align: center;
+    margin-right: 18px;
+    margin-top: -1px;
+}
+
+.small-team__text-intro {
+    opacity: 0.9;
+    font-family: Montserrat;
+    font-size: 38px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.05em;
+    color: #000000;
+    margin-left: 23px;
+    margin-bottom: 9px;
+}
+
+.small-team__text-main {
+    opacity: 0.61;
+    font-family: Merriweather;
+    font-size: 17px;
+    line-height: 28px;
+    font-weight: 400;
+    text-transform: uppercase;
+    letter-spacing: 0.072em;
+    color: #000000;
+    margin-left: 20px;
+}
+
+.small-team__images {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 116px;
+    justify-content: space-between;
+    row-gap: 14px;
+}
+
+.team-member_first {
+    position: relative;
+    height: 440px;
+    width: 370px;
+    background-image: url(../images/first_member.png);
+}
+
+.team-member_second {
+    position: relative;
+    height: 440px;
+    width: 370px;
+    background-image: url(../images/second_member.png);
+}
+
+.team-member_third {
+    position: relative;
+    height: 440px;
+    width: 370px;
+    background-image: url(../images/third_member.png);
+}
+
+.team-member_four {
+    position: relative;
+    height: 440px;
+    width: 370px;
+    background-image: url(../images/four_member.png);
+}
+
+.team-member_five {
+    position: relative;
+    height: 440px;
+    width: 370px;
+    background-image: url(../images/five_member.png);
+}
+
+.team-member_six {
+    position: relative;
+    height: 440px;
+    width: 370px;
+    background-image: url(../images/six_member.png);
+}
+
+.team-member_tooltip {
+    opacity: 0;
+    position: absolute;
+    bottom: 16px;
+    left: 21px;
+    width: 332px;
+    height: 241px;
+    padding: 40px 43px 42px 39px;
+    background-color: #71f6ec;
+    transition: .3s;
+    -webkit-transition: .3s;
+    -moz-transition: .3s;
+    -ms-transition: .3s;
+    -o-transition: .3s;
+}
+
+.team-member_tooltip::before {
+    content: '';
+    position: absolute;
+    top: -19px;
+    left: 9px;
+    width: 42px;
+    height: 37px;
+    background-image: url(../images/rect.png);
+    transition: .3s;
+    -webkit-transition: .3s;
+    -moz-transition: .3s;
+    -ms-transition: .3s;
+    -o-transition: .3s;
+}
+
+.team-member_tooltip:hover {
+    transition: .3s;
+    cursor: pointer;
+    opacity: 1;
+    transition: .3s;
+    -webkit-transition: .3s;
+    -moz-transition: .3s;
+    -ms-transition: .3s;
+    -o-transition: .3s;
+}
+
+.ho {
+    margin-right: 34px;
+    margin-left: -1px;
+}
+
+.ho:hover {
+    color: rgb(73, 28, 179);

```

Comment:
use css variables for colors, most popular colors initialize under the :root selector (https://css-tricks.com/almanac/selectors/r/root/)

Reply:
Thanks, I understand, but the colors are practically not repeated here. But in the future I will use it!